walletconnect-wallet-standard/src/index.js

```diff
@@ -1,69 +1,59 @@
-/* eslint-disable no-underscore-dangle, max-classes-per-file */
-import QRCodeModal from "@walletconnect/qrcode-modal";
+import {
+  SOLANA_DEVNET_CHAIN,
+  SOLANA_MAINNET_CHAIN,
+} from "@solana/wallet-standard-chains";
+import { WalletConnectModal } from "@walletconnect/modal";
 import WalletConnectClient from "@walletconnect/sign-client";
 import { getSdkError, parseAccountId } from "@walletconnect/utils";
-import base58 from "bs58";
+import { binary_to_base58 as binaryToBase58 } from "base58-js";
 
-// Errors
-// Taken from https://stackoverflow.com/a/41429145/2247097
-export class ClientNotInitializedError extends Error {
-  constructor() {
-    super();
+// need to redefine type because @implements doesnt allow import for some reason
+/** @typedef {import("@solana/wallet-standard-features").WalletWithSolanaFeatures} WalletWithSolanaFeatures */
 
-    // Set the prototype explicitly.
-    Object.setPrototypeOf(this, ClientNotInitializedError.prototype);
-  }
-}
+/** @typedef {Exclude<import("@solana/wallet-standard-chains").SolanaChain, "solana:localnet" | "solana:testnet">} NonLocalnetChain */
```

Comment:
```suggestion
/** @typedef {Exclude<import("@solana/wallet-standard-chains").SolanaChain, "solana:localnet" | "solana:testnet">} WcSupportedSolChain */
```
rename type for clarity